### ISSUE TITLE
Settings are not working at export

### ISSUE THREAD

**Issue by Tilix4** [superlou/fountain-atom#69]
Hi,

I've tried some settings for the export, and appart from "Print Profile" they seem not working.
Some I've tested : "Show page numbers"/ "Each Scene on new page" / "Footer" /  "Header" / "Watermark".

**Comment by jrbostic** [comment]
I'll take a look at it this weekend.

**Comment by Tilix4** [comment]
Thanks. By the way, I'll try to contribute during my August vacations.

**Comment by jrbostic** [comment]
Sorry, haven't been able to get to this yet.  Been in the middle of a big move... Thought I could do it sooner.  Will undertake to verify asap.

**Comment by jrbostic** [comment]
I've just run a test against this and found all of the settings under "Custom" section to be functional.  Have you seen any errors?  Any other information that could be related to producing this behavior?  The most likely way I could see this not rendering properly is if the config json isn't being properly written during a render... but I would expect some kind of write failure from the js code.  A file permission issue is always a possibility (for instance if the config file isn't owned by the current os user for some reason).

**Comment by thegreentimtam** [comment]
I'm having the same issue. No settings change anything in the PDF or previews.

**Comment by jrbostic** [comment]
@thegreentimtam @tillix4 Can we get some more information about your environments?  OS, OS version, Atom version, and Fountain version?  We have to figure out a way to reproduce this to fix it.

**Comment by thegreentimtam** [comment]
Windows 10 v1803; Atom 1.30.0; Fountain 0.8.0.

**Comment by Tilix4** [comment]
Hi, sorry for taking so long. I managed some tests with _outline-view-tests.fountain_ and tried to change settings like **Scene Numbers**, **Font Family**, **Double space between scenes**, and many other in random. I also tried in user and dev modes for each OS test:
Windows 10 v1803; Atom 1.30.0; Fountain 0.8.0 : Nothing changes.
OSX 10.11.6; Atom 1.30.0; Fountain 0.8.0 : Nothing changes.
KDE Neon 5.13; Atom 1.30.0; Fountain 0.8.0 : Nothing changes.

I compared exported PDF files with changed settings for every test, they are perfectly identicals. Not even the page format (a4 or us-letter) worked.
Hope it'll help.
